stable/anchore-admission-controller/templates/deployment.yaml

```
@@ -0,0 +1,68 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "anchore-admission-controller.fullname" . }}
+  labels: {{- include "anchore-admission-controller.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
```

since you're using the _newer_ way of doing helm labels, should this be something like
```suggestion
    matchLabels: {{- include "anchore-admission-controller.labels" . | nindent 6 }}
```

---

That will include a label which is not constant - and selectors are immutable AFAIK.